.pubnub.yml

```diff
@@ -399,13 +411,14 @@ supported-platforms:
   - 
     editors: 
       - "1.9.7"
-      - "1.10.3"
-      - "1.11.11"
-      - "1.12.6"
+      - "1.10.8"
+      - "1.11.13"
+      - "1.12.9"
+      - "1.13"
     platforms: 
       - "FreeBSD 8-STABLE or later, amd64, 386"
       - "Linux 2.6 or later, amd64, 386."
       - "Mac OS X 10.8 or later, amd64"
       - "Windows 7 or later, amd64, 386"
     version: "PubNub Go SDK"
-version: v4.2.7
+version: v4.2.8
```

Comment:
Should this be `v4.3.0` ? Just wondering since it's quite a big addition/change.

Reply:
done